src/webgpu/shader/validation/wgsl/variable-with-initilizer-module-workgroup.fail.wgsl

```diff
@@ -0,0 +1,8 @@
+# v-0032: variable 'w' has an initializer, however its storage class is 'workgroup'.
+
+var<workgroup> w : vec3<i32> = vec3<i32>(0, 1, 0);
```

Comment:
Is this a correct use of <wrokgoup> storage class or should the store type be 'array'?
Is there a way to initialize arrays in wgsl/tint?

Reply:
I believe this is correct.